tor/core/inbox.py

```diff
@@ -87,7 +87,10 @@ def process_reply(reply, config):
             reply.mark_read()
             return
 
-        if 'done' in reply.body.lower():
+        if (
+            'done' in reply.body.lower() or
+            'deno' in reply.body.lower()  # we <3 u/Lornescri
+        ):
```

Comment:
I was going to suggest we use a regex because it would be faster, but I just benchmarked it. Looks like running it with a regex is as much as 10x slower than 2 full text searches like this.

This should be fine. 👍